ui/StatusQ/src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -16,58 +15,210 @@ QtObject {
 
     property ThemePalette palette: StatusLightTheme {}
 
+    readonly property ThemePalette statusQLightTheme: StatusLightTheme {}
+    readonly property ThemePalette statusQDarkTheme: StatusDarkTheme {}
+
+    readonly property string sqAssetPath: Qt.resolvedUrl("../../../assets/")
+
+    function changeTheme(theme:int, isCurrentSystemThemeDark:bool) {
```

Comment:
Did JS add typed params or is that a QML thing?

Reply:
That's a QML thing, mostly a decoration for now but the qml compiler can use it and spot mistakes/mismatches